dist/_SassyCast.scss

```diff
@@ -1,188 +1,254 @@
+  $color-keywords: map-get($sc-constants, 'COLOR_KEYWORDS');
   $keywords: ();
 
   // Filling $keywords with stringified color keywords
-  @each $color in $colors {
+  @each $color in $color-keywords {
     $keywords: append($keywords, $color + '');
   }
 
-  // Deal with inherit keyword
-  @if ($value-lower == 'inherit') {
-    @return unquote($value);
-  }
-
-  // Deal with color keywords
+  // If the given value is a stringified color keyword, return the accurate
+  // color keyword.
   @if index($keywords, $value-lower) {
-    @return nth($colors, index($keywords, $value-lower));
+    @return nth($color-keywords, index($keywords, $value-lower));
   }
 
-  // Deal with hexadecimal triplets
+  // If the color starts with a hash, assume it’s a hexadecimal color triplet
+  // and try to convert it to an actual color.
   @else if (str-slice($value-lower, 1, 1) == '#') {
     @return _sc-from-hex($value);
   }
 
-  // Deal with rgb(a) colors
+  // If the color starts with `rgb`, assume it’s a rgb(..) function call and try
+  // to convert it to an actual color.
   @else if (str-slice($value-lower, 1, 3) == 'rgb') {
     @return _sc-from-rgb($value);
   }
 
-  // Deal with hsl(a) colors
+  // If the color starts with `hsl`, assume it’s a hsl(..) function call and try
+  // to convert it to an actual color.
   @else if (str-slice($value-lower, 1, 3) == 'hsl') {
     @return _sc-from-hsl($value);
   }
 
-  // Return value
-  @else {
-    //@warn 'Could not cast `#{inspect($value)}` to color.';
-    @return null;
+  // If everything has failed, arn and return the default color value (or throw
+  // in strict mode).
+  @return _sc-throw($value, 'color');
+}
+
+/// Helper function to throw when running in strict mode, or warn and return
+/// default value for expected type when running in non-strict mode.
+/// @param {*} $value
+/// @param {String} $expected-type
+@function _sc-throw($value, $expected-type) {
+  $return-value: map-get($sc-non-strict-defaults, $expected-type);
+
+  @if $sc-strict-mode {
+    @error 'Could not cast `#{inspect($value)}` to #{$expected-type}.';
+  } @else {
+    @warn 'Could not cast `#{inspect($value)}` to #{$expected-type}; returning `#{$return-value}`.';
   }
+
+  @return $return-value;
 }
 
-/// Convert to bool
+/// Convert to list
 /// @param {*} $value - value to cast
-/// @return {Bool}
-@function _sc-to-bool($value) {
-  @if (not $value or $value == '' or $value == 0) {
-    @return false;
+/// @return {List}
+@function to-list($value) {
+  $type: type-of($value);
+  
+  // If the value is already a list, we can safely return it.
+  @if ($type == 'list') {
+    @return $value;
   }
 
-  @return true;
+  // If the value is a map, we return a zip of keys and values (aka a list-map).
+  @if ($type == 'map') {
+    @return zip(map-keys($value), map-values($value));
+  }
+
+  // Else we wrap the item in a single-item list.
+  @return ($value,);
 }
```

Comment:
Probably on nothing, to be consistent with how other languages handle strings (as a list/array of characters)